modules/op-resolver-bundle.adoc

```diff
@@ -4,13 +4,13 @@
 // // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
 :_mod-docs-content-type: PROCEDURE
 [id="resolver-bundles-specify_{context}"]
-= Specifying a remote pipeline or task using the bundles resolver
+= Specifying a remote pipeline, task, or step action using the bundles resolver
```

Comment:
```suggestion
= Specifying a remote pipeline, task, or StepAction using the bundles resolver
```

Reply:
The style guide does not permit us to use StepAction as a standalone noun. It can be a StepAction definition, StepAction resource, and so on, but when it's a standalone thing, it's "step action".